app/services/catalog/create_request_for_applied_inventories.rb

```diff
@@ -0,0 +1,34 @@
+module Catalog
+  class CreateRequestForAppliedInventories
+    attr_reader :order
+
+    def initialize(id)
+      @order = Order.find_by!(:id => id)
+      @item = @order.order_items.first
+    end
+
+    def process
+      send_request_to_compute_applied_inventories
+
+      @order.update(:state => "Waiting for inventories")
```

Comment:
@mkanoor do you have any thoughts on what this state should be?

Reply:
Whatever that state ends up being it would need to be added to the openapi.json spec file as well.

Reply:
@syncrou yes we would have to add this to openapi.json. So when a order is Submitted the first state would be "Waiting for Inventories" before it transitions to the "Approval"

Reply:
I know it needs to be added but I wasn't sure if "Waiting for inventories" was an official state we wanted to use. Do we want to change this to different wording?

Reply:
We could keep the state as the "initializing" and just change the message to indicating that its waiting for inventories